app/src/main/java/com/example/concordia_campus_guide/Fragments/LocationFragment/LocationFragmentViewModel.java

```diff
@@ -52,14 +62,27 @@
     public static final Logger LOGGER = Logger.getLogger("LocationFragmentViewModel");
     public static final String FLOORS_AVAILABLE = "floorsAvailable";
 
+
+    private PolylineOptions displayedPolylineOption;
+    private Polyline currentlyDisplayedLine;
+    private HashMap<String, List<WalkingPoint>> walkingPointsMap = new HashMap<>();
+    private List<WalkingPoint> walkingPoints;
+
+    //Polyline styling
+    public static final int PATTERN_DASH_LENGTH_PX = 20;
+    public static final int PATTERN_GAP_LENGTH_PX = 20;
+    public static final PatternItem DASH = new Dash(PATTERN_DASH_LENGTH_PX);
+    public static final PatternItem GAP = new Gap(PATTERN_GAP_LENGTH_PX);
+    public static final List<PatternItem> WALK_PATTERN = Arrays.asList(GAP, DASH);
+
```

Comment:
Why don't put them in the constant values classes?

Reply:
I didn't know we had such a thing, we will update it.